.github/workflows/build.yaml

```diff
@@ -72,8 +72,8 @@ jobs:
       fail-fast: false
       matrix:
         cuda_version:
-          - '12.9.1'
-          - '13.0.0'
+          - *latest_cuda12
+          - *latest_cuda13
```

Comment:
GitHub just recently announced support for YAML anchors: https://github.blog/changelog/2025-09-18-actions-yaml-anchors-and-non-public-workflow-templates/

This seems like a useful application for them.